common-util/src/main/java/org/sunbird/common/responsecode/ResponseMessage.java

```diff
@@ -355,6 +356,13 @@
     String NO_EMAIL_RECIPIENTS =
         "Email notification is not sent as the number of recipients is zero.";
     String PARAMETER_MISMATCH = "Mismatch of given parameters: {0}.";
+    String ERROR_CONFIG_LOAD_EMPTY_STRING =
+        "Load configuration failed as empty string is passed as parameter.";
+    String ERROR_CONFIG_LOAD_PARSE_STRING = "Load configuration failed due to parsing error.";
+    String ERROR_CONFIG_LOAD_EMPTY_CONFIG = "Load configuration failed.";
+    String FIELD_CONFIGURED_IN_MULTIPLE_FIELDS_LISTS =
+        "{0} fields are configured in multiple fields lists.";
```

Comment:
Field {0} in {1} configuration is conflicting in {2} and {3}.

e.g. 
{0} = email
{1} = user
{2} = publicFields 
{3} = privateFields